cmd/registry/cmd/auth/token.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2022 Google LLC. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package auth
+
+import (
+	"fmt"
+
+	"github.com/apigee/registry/pkg/config"
+	"github.com/spf13/cobra"
+)
+
+func tokenCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "token",
+		Short: "Invoke the generator to print an auth token. See: `registry auth generator`.",
```

Comment:
This command is to allow other tools to get tokens, right? If it's analogous to `gcloud auth print-*-token` could we name it `registry auth print-token` (for consistency)?

Reply:
Sure.